compute_tools/src/pg_helpers.rs

```diff
@@ -480,18 +474,17 @@ pub async fn tune_pgbouncer(
         }
     };
 
-    if let Some(tls_config) = tls_config {
-        // pgbouncer starts in a half-ok state if it cannot find these files.
-        // It will default to client_tls_sslmode=deny, which causes proxy to error.
-        // There is a small window at startup where these files don't yet exist in the VM.
-        // Best to wait until it exists.
-        loop {
-            if let Ok(true) = tokio::fs::try_exists(&tls_config.key_path).await {
-                break;
-            }
-            tokio::time::sleep(Duration::from_millis(500)).await
-        }
```

Comment:
Removed as we wait for the TLS config on compute_ctl startup now instead. Although I think we do spawn the `tune_pgbouncer` fn before awaiting that task... hmm.